scripts/docker-integration-tests/carbon/test.sh

```diff
@@ -58,3 +73,18 @@ echo "foo.min.catch-all.baz 10 $t" | nc 0.0.0.0 7204
 echo "foo.min.catch-all.baz 20 $t" | nc 0.0.0.0 7204
 echo "Attempting to read mean aggregated carbon metric"
 ATTEMPTS=10 TIMEOUT=1 retry_with_backoff read_carbon foo.min.catch-all.baz 15
+
+t=$(date +%s)
+echo "a 0 $t"             | nc 0.0.0.0 7204
+echo "a.bar 0 $t"         | nc 0.0.0.0 7204
+echo "a.biz 0 $t"         | nc 0.0.0.0 7204
+echo "a.biz.cake 0 $t"    | nc 0.0.0.0 7204
+echo "a.bar.caw.daz 0 $t" | nc 0.0.0.0 7204
+echo "a.bag 0 $t"         | nc 0.0.0.0 7204
+ATTEMPTS=5 TIMEOUT=1 retry_with_backoff find_carbon a* a.json
```

Comment:
suggestion(take/leave): i think breaking the files out here actually makes it harder to follow along what you're trying to do. would read easier if everything was inline. e.g. --

```
ATTEMPTS=5 TIMEOUT=1 retry_with_backoff find_carbon a* <(cat <<EOF
[
  {
    "id": "a",
    "text": "a",
    "leaf": 0,
    "expandable": 1,
    "allowChildren": 1
  }
]
EOF)
```